spec/rails_helper.rb

```diff
@@ -8,3 +8,19 @@
 def jsonapi_response
   JSON.parse(response.body, symbolize_names: true)
 end
+
+def post_with_rails_fix(url, params: {}, **others)
+  if Rails::VERSION::MAJOR >= 5
+    post url, params: params, **others
+  else
+    post url, params, **others
+  end
+end
+
+def patch_with_rails_fix(url, params: {}, **others)
```

Comment:
Same as with `post_with_rails_fix` - some additional commented info would be great to understand why is this necessary and where does `others` come from